core/controllers/galleries.py

```diff
@@ -113,6 +113,10 @@ def get(self):
         explorations_list = (
             summary_services.get_displayable_exp_summary_dicts_matching_ids(
                 exp_ids))
+        # TODO(mgowano): Remove '' in query to allow for collection searches
+        collections_list = (
+            collection_services.get_displayable_collections_matching_query(
```

Comment:
To be consistent with the exploration summary retrieval above, you can do the cursor search for collections here. That should also resolve your TODO, since you can use the query right away.